app/styles/discord.css

```diff
@@ -6,6 +6,19 @@
   height: 100%;
 }
 
+.linking__logo {
+  margin-top: 20px;
+  display: flex;
+}
+
+.linking__logo>.line {
+  width: 300px;
+  border-top: none;
+  position: relative;
```

Comment:
Why using position relative here when parent is flex already?
We can use align-items in the parent to align it vertically and remove this stylings from here.

Reply:
because i want the line to move up relatively 50%